pkg/kubernetes/secrets/store.go

```diff
@@ -113,7 +114,7 @@ func (s *Store) Create(ctx context.Context, keyName string, keyValue string, val
 	data := map[string][]byte{
 		SecretDataKey: byteValue,
 	}
-	secret := &v1.Secret{ObjectMeta: metav1.ObjectMeta{Name: keyValue}, Immutable: &Immutable, Data: data}
+	secret := &v1.Secret{ObjectMeta: metav1.ObjectMeta{Name: strings.ToLower(keyValue)}, Immutable: &Immutable, Data: data}
```

Comment:
See my comment on the original issue #115 that we need to handle converting during resolve too.

Reply:
https://github.com/getporter/kubernetes-plugins/blob/dffd4ff062116a5127ba0c1129c97708ddb8d589/pkg/kubernetes/secrets/store.go#L77 
I think Resolve already handle this ?